src/Aquarius.ONE.ClientSDK/Operations/Sample/SampleCache.cs

```diff
@@ -272,4 +321,14 @@ private T ErrorResponse<T>(Exception exception, T result)
             return result;
         }
     }
+
+    public class SchedulePropertyBag
+    {
+        public TestGroup TestGroup { get; set; }
+    }
+
+    public class TestGroup
+    {
+        public Guid Id { get; set; }

```

Comment:
Is this and the TestGroup in the SchedulePropertyBag class above part of the cache? Should they have public setters?

Reply:
If it's private setters then it is not accessible from SampleCache. I want to use it to desearialize the property bag